### PR TITLE
Add reducers, action-creators, actions, api for browseSkills

### LINKED ISSUE #1742: Create api, reducer, action creator and action type for browseSkill components
**Actual Behaviour**

Create API using ajax, reducer, action creator and action type for browseSkill components

**Expected Behaviour**

Implement for redux integration

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1742 

Changes: 

- [x] Changed handleShowSkills, used simple logic to set checkboxes and removed showSkills. Both Checkbox filters now work independently(Show staff and show review skills). Instead used separate states i.e. **showReviewedSkills** and **showStaffPicks** Made the appropriate changes in API URL.

- [x] Added action creators for browseSkills
- [x] Reducers
- [x] Action types
- [x] API using ajax

Once this is done, we can start with https://github.com/fossasia/susi_skill_cms/issues/1741

Updated link: https://pr-1751-1-fossasia-susi-skill-cms.surge.sh

Surge Deployment Link: https://pr-1751-fossasia-susi-skill-cms.surge.sh

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd @anshumanv please review

File: src/redux/actions/skills.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import * as apis from '../../apis';
+
+const returnArgumentsFn = function(payload) {
+  return Promise.resolve(payload);
+};
+
+const skillActions = {
+  getMetricsSkills: createAction(
+    actionTypes.SKILLS_GET_MATRIX_SKILLS,
+    apis.fetchMatrixSkills,
```
Comment (akshatnitd) [inline src/redux/actions/skills.js]: Correct typo - `Metric`

File: src/components/BrowseSkill/BrowseSkill.js [hunk 2]
```diff
@@ -389,7 +369,10 @@ export default class BrowseSkill extends React.Component {
         orderBy +
         '&filter_type=' +
         filter +
-        showSkills;
+        '&reviewed=' +
```
[the remark this answers is not in the document]
Comment (fragm3) [reply src/components/BrowseSkill/BrowseSkill.js]: @akshatnitd Yes the URL will be created in api file, for now we have to still integrate redux. And connect with api, I will create a PR for https://github.com/fossasia/susi_skill_cms/issues/1741 , just for the time being, to keep the things working the URL is kept here

**Comment by fragm3** [comment]
@akshatnitd made the changes, updated link: https://pr-1751-1-fossasia-susi-skill-cms.surge.sh

**Comment by fragm3** [comment]
@akshatnitd resolved the conflicts, please review

**Pull request merged**